<launch>
    
    <include file="$(find freenect_launch)/launch/freenect.launch">
        <arg name="rgb_frame_id" value="/kinect2_rgb_frame" />
        <arg name="depth_frame_id" value="/kinect2_depth_frame" />
        <arg name="camera" value="kinect2" />
        <arg name="device_id" value="#2" />
        <arg name="depth_registration" value="true" />
    </include>
        
    <arg name="marker_size" default="13.35" />
    <arg name="max_new_marker_error" default="0.08" />
    <arg name="max_track_error" default="0.2" />
    <arg name="cam_image_topic" default="/kinect2/rgb/image_color" />
    <arg name="cam_info_topic" default="/kinect2/rgb/camera_info" />
    <arg name="output_frame" default="/kinect2_link" />
    <arg name="mname" default="kinect1" />
    
    <node name="kinect2_location_publisher" pkg="final" type="broadcast_kinect2.py" respawn="false" output="screen" /> 

</launch>
